src/ts/components/playbacktimelabel.ts

```diff
@@ -119,6 +119,8 @@ export class PlaybackTimeLabel extends Label<PlaybackTimeLabelConfig> {
     let playbackTimeHandler = () => {
       if (!live && player.getDuration() !== Infinity) {
         this.setTime(PlayerUtils.getCurrentTimeRelativeToSeekableRange(player), player.getDuration());
+      } else {
+        this.getDomElement().removeAttr('aria-label');
```

Comment:
Why do we remove it in case of `live` 🤔 

Please move this to another place as this is executed every `TimeChanged`.

EDIT: Do you need to remove it every time (in live) because the value is changing? (Can we update it)